examples/todo/js/subscriptions/AddTodoSubscription.js

```diff
@@ -12,11 +12,8 @@ export default class AddTodoSubscription extends RelaySubscriptions.Subscription
   getSubscription() {
     return Relay.QL`subscription {
       addTodoSubscription {
-        clientMutationId
         todoEdge {
-          __typename
           node {
-            __typename
```

Comment:
Okay :+1: The reason I had this was because the babel plugin didn't add this automatically for some reason.